src/os/filestore/FileStore.cc

```diff
@@ -107,6 +107,8 @@ using ceph::crypto::SHA1;
 #define XATTR_SPILL_OUT_NAME "user.cephos.spill_out"
 #define XATTR_NO_SPILL_OUT "0"
 #define XATTR_SPILL_OUT "1"
+#define __FUNC__ __func__ << "(" << __LINE__ << ")"
+//#define __FUNC__ __PRETTY_FUNCTION__ << "(" << __LINE__ << ")"
```

Comment:
Could you please remove the commented code ? This was not there when I approved the changes !!!